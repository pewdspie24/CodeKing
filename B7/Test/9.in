19
15 326646616
3731830 3170868
5008814 5800737
6552619 2417205
2375075 3582100
6166900 6678859
4436688 6050426
9339156 276759
1807556 916635
8672869 592425
6722930 2064275
8165580 3233321
8759484 6646396
4246420 4211401
5239360 9731688
9660371 5392178
26 296517200
1315306 351255
5351388 6038684
9097661 3648674
7003833 521097
7025021 9040076
3650394 4630287
4960811 6016086
1038042 2294500
1655438 5196358
2165335 733406
1995184 140967
339339 5537059
3670110 2326153
7399381 4153101
5836957 9767185
8222398 1549899
5512429 9459475
6519220 9071183
7163492 6924448
1502493 9303862
1028181 577678
5397234 3784084
9723204 5689667
4492165 9900027
1766157 1929580
1444629 2943405
10 878728472
6781503 8552778
6423456 5113558
2789102 1865766
8445131 8451399
5729858 5616308
4967288 1793730
3301432 7521392
3374678 9426186
6779777 6640086
7810115 5512445
30 900727306
222495 2382699
7268304 9995783
6028272 8475391
5220530 5979521
6370964 6515248
2480632 1600193
4016533 2849381
9918847 9652716
8394563 894958
3748533 9362994
1877181 5046787
7718428 3995125
2736160 6565882
4772275 3522033
1501119 8993638
1382563 7137859
7964158 4150643
231866 7279727
9705897 4542445
7496090 3641379
7793179 8569042
3263278 1116904
195344 5750357
2147125 4384797
479299 8974694
7743199 2823982
4791889 9522536
8017930 2713808
6771097 1373590
5526409 3122624
29 293613247
1159407 4709839
2963990 3666832
3131115 2493355
1183274 9135439
5581811 2314845
4514834 7003527
2055935 6059479
245567 1146226
1753856 171550
2734537 7723651
4751202 702042
9418575 1369417
8717025 3125192
3861362 6438414
3655610 3730647
4150611 1200401
158707 2379176
579989 9492144
3887498 5659232
8046338 7032578
2032929 3304033
164906 1156137
5513676 6806239
7663345 4504119
9892454 110089
2791329 326911
3641284 554767
2590734 9031046
5056184 5103752
18 451231844
2817392 8711202
9197144 6159824
3058450 2846474
4466415 9445313
2890187 8459674
7948686 1348592
9416767 2991935
5595047 5420428
3575567 1373654
7774587 3155368
9593926 5646565
8963253 8507207
6990681 4546293
3029221 8331994
4355817 5391698
8164291 8141412
3499120 4708764
3666189 5050353
22 745361685
2153387 1311243
3459333 5650322
9749406 9266020
5970362 5481497
6054017 616302
2271846 9008078
7084948 7674519
861576 570229
8863374 3224004
1880739 6587852
1248929 9445204
9420285 8573835
9952682 4012192
5897337 1773206
436872 8898725
1402602 2344205
6010843 9951419
1682405 1194443
2827530 147472
1579474 629941
6265082 5926751
212586 442089
15 746629674
1731296 6590361
3709004 511571
2341628 2265907
1237199 2903735
9872955 1469487
6218053 6919958
4631166 8593921
843880 8344057
1265447 8066308
5818026 1796655
9838952 3579726
2061182 1846532
726129 1111911
8446971 2625552
8931718 6314145
29 777277899
6797959 6824238
1264861 4568640
9654179 9700294
638089 8333849
2494586 7406865
4179391 1570037
9825043 1795470
2013315 1057264
4774481 9806147
2727407 3758519
8784810 838061
3507490 2505263
7247608 4738657
8825944 9538645
6379610 5277651
4976103 1515148
5059015 5479838
1231849 9732873
5841446 8133655
5828683 1781591
7969976 449878
7185011 8544003
5632932 1918949
5732239 1122185
465226 4033022
3884831 5687980
6840618 6889802
565728 9649634
6381761 2042827
13 79631105
8074812 6363628
7278725 8264398
6947605 690776
3376747 5155542
420666 6401270
7276760 2381414
806812 9512303
8213692 7299023
8793553 5680434
3422009 4004865
8894163 4903895
6944727 4443741
8302574 5372398
13 161026196
8998492 6357216
1181093 8824615
1717020 5709799
3868818 1181929
4263988 6322487
259391 8823717
1756637 5083555
3163498 4793088
5429787 4100300
8857752 6917170
3686291 611815
7129858 8581603
1059942 8494367
23 680486998
9092627 8055926
9550394 5494445
1019162 1330805
2369400 1347628
3911275 6219837
4202868 2434465
5186346 4752822
851850 5368838
9635964 6616174
282580 6345101
2547437 9039849
3237735 6347542
355240 7426277
1851685 4454497
3628664 105981
9655338 9403063
1382015 9216722
3456651 4288812
510446 8309271
653275 6345920
6694781 8021892
9465680 2880449
8939587 7165710
30 83879633
3177368 1612464
7545529 936906
2687288 7844884
4889603 3291409
3776643 1473922
6086503 726777
1498829 719700
954455 5029474
242999 2485813
5097349 7636541
7368580 7073000
4079820 4519045
6739022 4867839
2265891 5238767
1444517 2235028
2264412 4659931
4857299 5979005
4941210 7748263
4447656 9437604
4771234 9217262
726504 8392823
8833640 581014
2531869 7911360
6870934 1488023
7113320 2201208
8565550 4114161
4838968 4826662
1977545 8701850
1586762 8700580
2584429 8374529
13 724171460
3433052 328589
7523748 4475187
7023814 6619200
9649869 4467559
3563304 998007
6319946 8256473
5452135 8242816
9723855 2568154
9475888 5757242
6256771 5136948
801942 6300000
4771504 6229209
6943663 8543459
20 129618763
4069690 4504305
1320484 7914112
2629114 4650287
3888442 9311171
4643629 8690256
6037698 2463150
711396 1096869
9484951 1432502
203737 9056055
8525230 1797297
565587 6211458
863380 9467074
3909835 8899947
7772070 8413641
1419293 3018079
8834408 1835066
3277934 4563967
9281537 772497
4653996 1270464
976345 1494010
28 283542706
5647037 3062196
4743452 3423322
9256716 6126100
7260132 3350181
4142280 132110
8616601 7991221
2343203 7662241
3695790 212117
8332246 2234739
2114017 6691666
9439188 8893082
6544096 6362543
668895 7369551
756559 5295709
1416909 7891358
7013124 2484671
3972302 3631995
5487835 5508036
1225402 9718984
4932945 9509689
7030480 1127450
5834415 6397849
888878 7960901
5704965 6896995
9640160 6803513
6865385 6233380
6132708 4292183
9093888 1627938
16 55201335
2434315 5541170
3504105 4235973
7128203 6389873
3045390 4507942
3402176 2320338
4214189 121509
9449580 5534902
5280924 7476533
9594771 2994206
1454342 7735930
1568832 4852376
7834759 9269770
124610 2565794
1464804 3747481
915337 2874299
6499900 9672950
23 863010991
4174286 1219454
974307 1793132
5913477 1966646
7154725 611185
9653203 1763511
2791537 3062107
6874200 9295317
8688937 2864261
3971484 4308986
6322986 1958986
4076665 7817817
666465 7396101
5778989 6910969
2032289 9384096
1558648 130702
474907 5771803
2630674 2603579
4527252 2256562
2975423 7078055
1587033 7033226
7494071 110660
6161721 8331629
8116855 4494082
10 265448553
8935511 5816535
1452524 2520439
1307666 6910716
5448383 2717955
8340857 8257205
9540990 2476026
5432185 7928766
3360181 9746169
4489810 1241817
2887415 6581889
