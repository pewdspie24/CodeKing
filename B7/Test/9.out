36
44
276
120
31
68
125
89
97
13
51
103
11
101
16
38
8
110
38
